src/tests/secp256k1_tests.rs

```diff
@@ -253,6 +253,37 @@ fn verify_invalid_signature() {
     assert!(kp.public().verify(&digest.0, &signature).is_err());
 }
 
+#[test]
+fn fail_to_verify_if_upper_s() {
```

Comment:
Why do we need this here? I thought we've already added that functionality + this PR should be HKDF specific.

Reply:
sorry, did a merge instead of rebase with main, reverted